AElf.Sdk.CSharp/Context.cs

```diff
@@ -55,7 +55,7 @@ public void FireEvent(Event logEvent)
         public Hash TransactionId => TransactionContext.Transaction.GetHash();
         public Address Sender => TransactionContext.Transaction.From.ToReadOnly();
         public Address Self => SmartContractContext.ContractAddress.ToReadOnly();
-        public Address Genesis => Address.Genesis.ToReadOnly();
+        public Address Genesis => Address.Genesis;
```

Comment:
why remove to readonly?

Reply:
Because Address.Genesis is already read only.